integrations-and-supported-tools/tensorboard/notebooks/Neptune_Tensorflow_Tensorboard.ipynb

```diff
@@ -0,0 +1,453 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "GG3_2HyRVY4Y",
+    "tags": [
+     "header",
+     "comment"
+    ]
+   },
+   "source": [
+    "# Neptune + Tensorboard (Tensorflow)\n",
+    "\n",
+    "Introduction\n",
+    "\n",
+    "This guide will show you how to:\n",
+    "\n",
+    "* Use `enable_tensorboard_logging` to log metrics and metadata to both Tensorboard and Neptune"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "FUaDrmRu-h3k"
+   },
+   "source": [
+    "## Before you start\n",
+    "\n",
+    "This notebook example lets you try out Neptune anonymously, with zero setup.\n",
+    "\n",
+    "If you want to see the example logged to your own workspace instead:\n",
+    "\n",
+    "  1. Create a Neptune account. [Register &rarr;](https://neptune.ai/register)\n",
+    "  1. Create a Neptune project that you will use for tracking metadata. For instructions, see [Creating a project](https://docs.neptune.ai/setup/creating_project) in the Neptune docs."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "w_0KXqlePqNi"
+   },
+   "source": [
+    "## Install Neptune and dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "i36OvZAgvxkj"
+   },
+   "outputs": [],
+   "source": [
+    "%pip install -U neptune neptune-tensorboard==1.0.0rc2 tensorflow --no-cache-dir"
```

Comment:
Will update to `neptune[tensorboard]` once https://github.com/neptune-ai/neptune-client/pull/1368 is in

Reply:
Thanks!
Will merge this now, but will need to update the install to `neptune[tensorboard]` once the client is updated